hack/build-local-images.py

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/env python
+
+import sys
+
+from os import getenv, mkdir
+from os.path import abspath, dirname, join
+
+from tempfile import mkdtemp
+from shutil import copyfile, rmtree
+from subprocess import call
+
+image_config = {
+	"openshift/origin": {
+		"directory": "images/origin",
+		"binaries": {
+			"openshift": "/usr/bin/openshift"
+		},
+		"files": {}
+	},
+	"openshift/origin-deployer": {
```

Comment:
you can't hardcode the origin prefix, this should work when i'm using it with ocp too.